Pipfile.lock

```diff
@@ -1629,6 +1656,14 @@
             ],
             "version": "==1.7.1"
         },
+        "pipenv": {
+            "hashes": [
+                "sha256:56ad5f5cb48f1e58878e14525a6e3129d4306049cb76d2f6a3e95df0d5fc6330",
+                "sha256:7df8e33a2387de6f537836f48ac6fcd94eda6ed9ba3d5e3fd52e35b5bc7ff49e",
+                "sha256:a673e606e8452185e9817a987572b55360f4d28b50831ef3b42ac3cab3fee846"
+            ],
+            "version": "==2018.11.26"
+        },
```

Comment:
Is that safe to use pipenv 2018.11.26? What do you think @jma ?

I remember that we have problems with Python >= 3.6.4, do you confirm?